src/EtherFiNode.sol

```diff
@@ -192,6 +192,16 @@ contract EtherFiNode is IEtherFiNode {
         pod.requestWithdrawal{value: msg.value}(requests);
     }
 
+    //-------------------------------------------------------------------
+    //-------------  Execution-Layer Triggered Consolidations  ----------
+    //-------------------------------------------------------------------
+    function requestConsolidation(IEigenPod pod, IEigenPod.ConsolidationRequest[] calldata requests) external payable {
```

Comment:
you should only ever get the pod via node.getEigenPod(). Not pass it in as a param

Reply:
I also think you should drop that param from the other PR as well